core/src/stores/postgres.rs

```diff
@@ -39,6 +39,23 @@ use crate::{
     utils,
 };
 
+#[derive(Debug)]
+enum TablePrefix {
+    DataSourceDocument, // "dsd."
+    Table,              // "t."
+    Folder,             // "dsn."
+}
+
+impl TablePrefix {
+    fn as_str(&self) -> &'static str {
+        match self {
+            TablePrefix::DataSourceDocument => "dsd.",
```

Comment:
This is also quite brittle as any change to the name in the query will break that logic. 

I thiiiiiink we'd be better off with &str table names but we likely need two given the logic you're introducing?

Reply:
yes we'll need at least the two table names. At first I actually passed the 3 prefixes - maybe passing the 3 full column names would be more clear.